magda-scala-common/src/main/scala/au/csiro/data61/magda/model/Misc.scala

```diff
@@ -92,7 +91,7 @@ package misc {
       quality: Double,
       hasQuality: Boolean = false,
       source: Option[DataSouce] = None,
-      creation: Option[DcatCreation] = None,
+      creation: Option[Provenance] = None,
```

Comment:
If the key `creation` still stay unchanged, then search API will response `dataset` object with key `creation` rather then `provenance`. Then, in `DatasetSummary` we probably can't reference `dataset.provenance.isOpenData`